src/components/SidebarItem.js

```diff
@@ -0,0 +1,40 @@
+import './SidebarItem.css';
+import {useState} from 'react';
+import DeleteIcon from '@mui/icons-material/Delete';
+import IconButton from '@mui/material/IconButton';
+import DeletePopup from './DeletePopup';
+
+export default function SidebarItem (props) {
+	const [showDeletePopup, setShowDeletePopup] = useState(false);
+
+	function toggleModal() {
+		setShowDeletePopup(!showDeletePopup);
+	}
+
+	return (
+		<div>
+			
+			<li className="menu-item">
+				<label className='radio-button'>
+					<input type="radio" name="sidebar-item" 
+						checked = {props.list.id === props.currentListId? true : false}
```

Comment:
Just:
props.list.id === props.currentListId
not:
props.list.id === props.currentListId? true : false